src/SIUToba/rest/http/vista_respuesta.php

```diff
@@ -24,21 +24,23 @@ abstract public function get_cuerpo();
 
     protected function escribir_encabezados()
     {
-        $status = $this->respuesta->get_status();
-        //Send status
-        if (strpos(PHP_SAPI, 'cgi') === 0) {
-            header(sprintf('Status: %s', respuesta::getMessageForCode($status)));
-            //echo sprintf('Status: %s', self::getMessageForCode($this->status));
-        } else {
-            header(sprintf('HTTP/%s %s', '1.1', respuesta::getMessageForCode($status)));
-            //echo sprintf('HTTP/%s %s', '1.1', self::getMessageForCode($this->status));
-        }
+        if (! headers_sent()) {    
```

Comment:
```suggestion
        if (headers_sent()) {
            return;
        }    
```

menos complejo y nos permite mandar al log un warning si eso ocurre y representa algo anormal?